src/app/Scenes/MyCollection/Components/MyCollectionBottomSheetModals/MyCollectionBottomSheetModalArtistPreview.tsx

```diff
@@ -67,6 +57,26 @@ export const MyCollectionBottomSheetModalArtistPreview: React.FC<
     [isPrivate]
   )
 
+  const dismissBottomView = () => {
+    setViewKind({ viewKind: null })
+  }
+
+  if (!artist || !me || !me.userInterest) {
+    return (
+      <AutoHeightBottomSheet
+        visible={!artist || !me || !me.userInterest}
```

Comment:
Is this safeguard needed? Because we pass the same condition to `visible` 🤔
```suggestion
    return (
      <AutoHeightBottomSheet
        visible={!artist || !me?.userInterest}
```